dips/dip-10.md

```diff
@@ -101,130 +110,55 @@ struct DiemIdDomainEvent {
-
-All fields, except for `diem_id`, in this object are optional - VASP can choose to share either of those fields or none.
-
-Additionally, all fields in this object are informational - **they must not be used for KYC and only intended for UI purposes**.
-```json
-{
-   "diem_id": "<Diem_ID>",
-   "display_name": "<Display name>",
-   "profile_picture": "<profile_picture_uri>"
+If the amount is below the travel rule limit, the sending VASP can send a p2p transaction with PaymentMetadata and the `reference_id` VASPs agreed on to settle the transaction. 
+```
+enum PaymentMetadata {
+    PaymentMetadataV0(ReferenceId),
 }
+type ReferenceId = [u8, 16];
 ```
 
-| Field 	      | Type  | Required? 	| Description 	           |
-|-------	      |------ |-----------	|-------------	           |
-|diem_id         |str    | Y           | Diem ID of the user |
-|display_name     |str    | N           | Display name of the user. Usually a full name |
-|profile_picture  |str    | N           | Profile picture URL(see below) |
-
-If a user does not exist, VASP can choose to return an HTTP error, or the empty User Object, depending on it’s privacy choices.
+If the amount exceeds the travel rule limit, VASP must trigger an off-chain protocol for KYC exchange. The same `reference_id` must be used to perform a TR as described in [DIP-1](https://github.com/diem/dip/blob/master/dips/dip-1.md).
 
-## Profile Picture URI
-Profile picture URI is a URI that represents a profile picture of a user. Wallet app must support two possible URI schemas for profile picture.
 
-* **Https schema.** If URI starts with https:, wallet must attempt to fetch profile picture from the HTTP URI provided. When fetching this URI wallet app will not provide any authentication. Wallet app will use GET method to fetch the profile picture specified using https schema.
-  * Ex.: *https://novi.com/pictures/alice.png*
-* **Data schema.** If URI starts with data:, wallet must interpret the URI as a [Data URL](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs) that contains profile picture encoded.
-  * Ex.: *data:image/gif;base64,AAAAAAAA*
-* **Unencrypted http** schema must not be used to share profile pictures.
+# Diem ID or Subaddress?
+[DIP-5](https://github.com/diem/dip/blob/master/dips/dip-5.md) defines subaddresses as a way of defining a user address. Between subaddress and Diem ID, the recommended way of representing a user address is to use a Diem ID, as Diem ID provides clear benefits over subaddresses:  
+* Subaddresses are ideally one-time use addresses, and need to be refreshed for each payment. However, a Diem ID can be shared once and will be a persistent identifier from a user's perspective.
+* There are concerns around leakage of PII when using subaddresses, because the subaddress has to be included in the on-chain transaction metadata. Diem ID provides better privacy, because Diem ID is exchanged off-chain.
```

Comment:
interesting... I'm not sure this is necessary, we're not writing an editorial, maybe strike and save for the blog update :P